release/edgex-taf.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 name: 'edgex-taf'
-version: '2.2.0'
-releaseName: 'kamakura'
-releaseStream: 'main'
+version: '2.3.0'
+releaseName: 'levski'
+releaseStream: 'levski'
```

Comment:
Should this be main?

Reply:
@cloudxxx8 can you confirm we want to release from the `levski` branch and not `main`?  Typically we release from `main` unless it's an LTS release. Typically we've been merging the test code back to `main` and tagging it on main

Reply:
@bill-mahoney @ernestojeda we did it wrong before.  for the previous release, we retagged to the karakuma branch eventually.
no mater it's LTS release, we have to open a branch for test script, because the main branch should alwasy test against main images.
long time ago, we did the following steps:
1. change main branch to test against the released images
2. tag main branch
3. change back to test against main images
the steps is redundant, and we might easily make mistakes in such short window,  in the end, we open branches for all the releases in the test script.